jwst/photom/photom.py

```diff
@@ -618,50 +624,69 @@ def photom_io(self, tabdata):
         -------
 
         """
-        # Get the conversion factor from the PHOTMJSR column of the table row
-        conv_factor = tabdata['photmjsr']
+        # Get the scalar conversion factor from the PHOTMJSR column of the table row
+        conversion = tabdata['photmjsr']
 
-        # Store the conversion factors in the meta data
-        log.info('PHOTMJSR value: %g', conv_factor)
+        # Store the conversion factor in the meta data
+        log.info('PHOTMJSR value: %g', conversion)
         if isinstance(self.input, datamodels.MultiSlitModel):
             self.input.slits[self.slitnum].meta.photometry.conversion_megajanskys = \
-                conv_factor
+                conversion
```

Comment:
Does this work? Do slits have `meta` attribute?

Reply:
Yes, the ``photometry.schema`` that's inherited by ``slitdata.schema`` has a meta object.